Source/DafnyCore/AST/DafnyAst.cs

```diff
@@ -77,108 +78,114 @@ public abstract class INode {
     // TODO: Re-add format tokens where needed until we put all the formatting to replace the tok of every expression
     internal IToken[] FormatTokens = null;
 
-    public RangeToken RangeToken {
-      set => rangeToken = value;
-    }
+    [DebuggerBrowsable(DebuggerBrowsableState.Never)]
```

Comment:
Why do we need this attribute?

Reply:
Because there is a field `rangeToken` so `RangeToken` is a duplicate normally.

Reply:
`rangeToken` is not shown in the debugger because it's non-public, so I think we can leave this attribute out.